packages/apidom-ls/src/config/common/schema/lint/type--equals-openapi-3-1--asyncapi-2.ts

```diff
@@ -9,7 +9,8 @@ import { OpenAPI31 } from '../../../openapi/target-specs.ts';
 const typeEqualsOpenAPI3_1__AsyncAPI2Lint: LinterMeta = {
   code: ApilintCodes.SCHEMA_TYPE,
   source: 'apilint',
-  message: 'type must be one of allowed values',
+  message:
+    'should be equal to one of the allowed values allowedValues: boolean, object, array, number, string, integer null',
```

Comment:
Missing comma here and in tests:
```suggestion
    'should be equal to one of the allowed values allowedValues: boolean, object, array, number, string, integer, null',
```